web/templates/attendance/index_coordinator.html.eex

```diff
@@ -32,8 +32,18 @@
             >
               <i class="material-icons">more_vert</i>
             </button>
-            <ul class="mdl-menu mdl-menu--bottom-right mdl-js-menu"
-                for="tr_menu_<%= offered_course.id %>"
+            <ul
+              class="
+                mdl-menu mdl-js-menu
+                <%=
+                  if index > 10 && index > length(@offered_courses)-4 do
```

Comment:
I don't exactly get what this condition do. would you be interested to extract the condition to a function that describes it's purpose?